copy/copy.go

```diff
@@ -83,10 +86,17 @@ func (d *digestingReader) Read(p []byte) (int, error) {
 type Options struct {
 	RemoveSignatures bool   // Remove any pre-existing signatures. SignBy will still add a new signature.
 	SignBy           string // If non-empty, asks for a signature to be added during the copy, and specifies a key ID, as accepted by signature.NewGPGSigningMechanism().SignDockerManifest(),
+	ReportWriter     io.Writer
 }
 
 // Image copies image from srcRef to destRef, using policyContext to validate source image admissibility.
 func Image(ctx *types.SystemContext, policyContext *signature.PolicyContext, destRef, srcRef types.ImageReference, options *Options) error {
+	if options.ReportWriter == nil {
+		options.ReportWriter = ioutil.Discard
+	}
```

Comment:
Oops, this is actually incorrect because options is a pointer; we either need to make a copy of `options` or add a separate `reportWriter` variable.